mne/tests/test_read_neuromag_selection.py

```diff
@@ -22,30 +22,30 @@ def test_read_selection():
 
     raw = read_raw_fif(raw_fname)
     for i, name in enumerate(sel_names):
-        sel = read_selection(name)
+        sel = read_vectorview_selection(name)
         assert ch_names[i] in sel
-        sel_info = read_selection(name, info=raw.info)
+        sel_info = read_vectorview_selection(name, info=raw.info)
         assert sel == sel_info
 
     # test some combinations
-    all_ch = read_selection(['L', 'R'])
-    left = read_selection('L')
-    right = read_selection('R')
+    all_ch = read_vectorview_selection(['L', 'R'])
+    left = read_vectorview_selection('L')
+    right = read_vectorview_selection('R')
 
     assert len(all_ch) == len(left) + len(right)
     assert len(set(left).intersection(set(right))) == 0
 
-    frontal = read_selection('frontal')
-    occipital = read_selection('Right-occipital')
+    frontal = read_vectorview_selection('frontal')
+    occipital = read_vectorview_selection('Right-occipital')
     assert len(set(frontal).intersection(set(occipital))) == 0
 
     ch_names_new = [ch.replace(' ', '') for ch in ch_names]
     raw_new = read_raw_fif(raw_new_fname)
     for i, name in enumerate(sel_names):
-        sel = read_selection(name, info=raw_new.info)
+        sel = read_vectorview_selection(name, info=raw_new.info)
         assert ch_names_new[i] in sel
 
-    pytest.raises(TypeError, read_selection, name, info='foo')
+    pytest.raises(TypeError, read_vectorview_selection, name, info='foo')
```

Comment:
if you want to go the extra mile, I usually modernize `pytest.raises` calls in tests that I'm updating/changing. That way someday in the distant future all of them will be updated